includes/class-sensei-quiz.php

```diff
@@ -1458,6 +1556,23 @@ public static function the_user_status_message( $quiz_id ) {
 		echo $message;
 	}
 
+	/**
+	 * Adds the quiz hidden fields.
+	 *
+	 * @since 3.15.0
+	 */
+	public static function the_quiz_hidden_fields() {
```

Comment:
This method renders all the hidden fields at the beginning of the form instead of inside the questions loop. This is necessary because, when paginated, not all questions are rendered, so not all hidden fields are present.